tests/unit/htdocs/modules/protector/Protector364RcFixesTest.php

```
@@ -0,0 +1,492 @@
+        //   - HTTP_X_SQL_PROBE  (arbitrary custom header via HTTP_* expansion)
+        //   - HTTP_USER_AGENT   (stock header)
+        //   - HTTP_FORWARDED    (RFC 7239 header)
+        //   - CONTENT_TYPE      (Content-Type header; not in HTTP_* namespace)
+        //   - CONTENT_LENGTH    (Content-Length header; not in HTTP_*)
+        //   - PHP_AUTH_USER     (Basic auth username)
+        //   - QUERY_STRING      (= $_GET serialized; already scanned via $_GET)
+        //   - REQUEST_URI       (URL path + query)
+        //   - PATH_INFO         (URL segment)
+        // None of them must appear in _dblayertrap_doubtfuls after the scan.
+        require_once XOOPS_PATH . '/modules/protector/class/protector.php';
+        $protector = \Protector::getInstance();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Protector' in method 'fix15_behaviouralRequestMetadataDoesNotPoisonDoubtfuls'. (undefined)

(StaticAccess)